### DISCUSSION TITLE
Wilderness Permissions

### DISCUSSION THREAD

**Discussion by ocsecnarf9253** [TownyAdvanced/Towny#6660] [Q&A] [answered]
How do I allow server members to break and place blocks in the wilds?

**Comment by Warriorrrr** [comment, marked as answer]
See the [faq](https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions), this is also covered in our [installation guide](https://github.com/TownyAdvanced/Towny/wiki/Installation).

**Reply by ocsecnarf9253** [reply]
ok, thank you very much
